src/components/toc.tsx

```diff
@@ -2,15 +2,15 @@ import React from 'react'
 import styled from 'styled-components'
 import { stringify } from '../utils/stringify'
 
-const ChapterTitle = styled.h1`
+const ChapterTitle = styled.div`
```

Comment:
@2color - Shouldn't this be `h2` ? 🤔

Reply:
See Joël's [comment](https://github.com/prisma/docs/pull/906#issuecomment-707217532).